packages/discord-bot/src/commands/praiseAdmin.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  SlashCommandBuilder,
+  SlashCommandSubcommandBuilder,
+} from '@discordjs/builders';
+import logger from 'jet-logger';
+import { announcementHandler } from '../handlers/announce';
+import { Command } from '../interfaces/Command';
+
+export const praiseAdmin: Command = {
+  data: new SlashCommandBuilder()
+    .setName('admin')
+    .setDescription('Commands to perform admin actions for Praise')
+    .addSubcommand(
+      new SlashCommandSubcommandBuilder()
+        .setName('announce')
+        .setDescription("Automatically announce messages in Users' DMs")
+        .addStringOption((option) =>
+          option
+            .setName('message')
+            .setDescription(
+              'The message content that you want to send in the DMs'
```

Comment:
"The message content to publish."

Reply:
how about "The message content to announce."?